tests/unit/test-extension-runners/test.chromium.js

```diff
@@ -356,10 +358,13 @@ describe('util/extension-runners/chromium', async () => {
     await runnerInstance.exit();
   });
 
-  it('does pass a user-data-dir flag to chrome', async () => {
+  it('does pass user-data-dir flag to chrome', async () => {
+
+    mock({}); // empty file system
```

Comment:
Just use the `withTempDir` utility + fs to create the temporary files in the tests, instead of depending on `mock-fs`.

Reply:
done. also removed mock-fs dependency